packages/cli/package.json

```diff
@@ -31,6 +31,7 @@
     "/lib"
   ],
   "dependencies": {
+    "@blitzjs/gui": "0.13.0",
```

Comment:
Even though I added it as a cli's dependency it is not working yet. When I run `blitz gui` the `spawn` is unable to find the package, it should be carried over during installation.
```bash
blitz gui                 
You are using alpha software - if you have any problems, please open an issue here:
  https://github.com/blitz-js/blitz/issues/new/choose
You started the Blitz GUI
{
  error: Error: spawnSync @blitzjs/gui ENOENT
  ...
```
So there must be something else I'm missing here 🤔